src/commands/commands/agile/commands/init/agileInit.test.ts

```diff
@@ -0,0 +1,7 @@
+import { agileInit } from "./agileInit";
```

Comment:
There are two standards that I know of, both of which are illustrated in the [jest config defaults for test matching](https://jestjs.io/docs/en/configuration#testmatch-arraystring). One, you have a directory `__tests__` that contains all tests. The other is to have all file names with their corresponding `file.test.(j|t)s`. I prefer option 2 because it makes it easier to see/discover tests. It should also be noted that none of the tests get packaged up into the `lib` itself. I'm totally open to other options too though